sycl/test-e2e/Matrix/Inputs/joint_matrix_bfloat16_packedB_impl.hpp

```diff
@@ -1,109 +1,128 @@
-//=----- joint_matrix_bfloat16_packedB_impl.hpp - DPC++ joint_matrix -------=//
+//===---joint_matrix_bfloat16_packedB_impl.hpp - DPC++ joint_matrix----------------===//
```

Comment:
in general `sycl/test-e2e/Matrix/Inputs/joint_matrix_bfloat16_impl.hpp` and `sycl/test-e2e/Matrix/Inputs/joint_matrix_bfloat16_packedB_impl.hpp` are almost identical and can be unified into 1 single test with matrix B layout being a parameter.
It should make maintenance easier.
Even during code review I noticed similar problems in both tests, so if it is a single test, we could fix the problems 1 time, instead of 2.